### DISCUSSION TITLE
working before using blackhole 2 ch for Mac 11

### DISCUSSION THREAD

**Discussion by phpnukes** [ExistentialAudio/BlackHole#400] [Q&A]
Even installed latest ch16 version. same issue unable adjust Mac audio devices channel volume. master Levels grey. Output volumes adjustable ok

Using with audacity to record computer sound card.

Month ago working perfectly tried all options in audacity preferences, mic and output nothing.

The only pointer is unable to Mac audio devices channel volume

**Comment by devinroth** [comment]
I disabled input volume control. There is output volume control.

They literally control the same thing.

**Comment by phpnukes** [comment]
Thank you.
RESOLVED
You are not going to believe what I was doing wrong.

Trying to set a new output device while I should have used the built in one.
Only need to set the input to BlackHole 16ch
Output ignore just use Built-in Output on what ever software you are using.

I'm using Audacity to record off Spotify
Mic selected aggregated device
Speaker selected Built-in

Select BlackHole only in aggregate device.
Ignore multi output device unless you need to.

Thank you Devin for all your hard work and efforts BlackHole is fab.
